datalad/config.py

```diff
@@ -118,7 +147,7 @@ def parse_gitconfig_dump(dump, cwd=None, multi_value=True):
         # line is a null-delimited chunk
         k = None
         # in anticipation of output contamination, process within a loop
-        # where we can reject non syntax compliant pieces
+        # scope we can reject non syntax compliant pieces
```

Comment:
yet another victim in text of the replace. TODO:
```suggestion
        # where we can reject non syntax compliant pieces
```